includes/embeds/class-amp-twitter-embed.php

```diff
@@ -229,15 +229,31 @@ private function create_amp_twitter_and_replace_node( $dom, $node ) {
 			return;
 		}
 
+		$additional_attributes = [];
+
+		if ( $node->hasAttributes() ) {
+			foreach ( $node->attributes as $attr ) {
+				$name  = $attr->nodeName;
+				$value = $attr->nodeValue;
+
+				if ( substr( $name, 0, 5 ) === 'data-' ) {
+					$additional_attributes[ $name ] = $value;
+				}
+			}
+		}
+
 		$new_node = AMP_DOM_Utils::create_node(
 			$dom,
 			$this->amp_tag,
-			[
-				'width'        => $this->DEFAULT_WIDTH,
-				'height'       => $this->DEFAULT_HEIGHT,
-				'layout'       => 'responsive',
-				'data-tweetid' => $tweet_id,
-			]
+			array_merge(
+				[
+					'width'        => $this->DEFAULT_WIDTH,
+					'height'       => $this->DEFAULT_HEIGHT,
+					'layout'       => 'responsive',
+					'data-tweetid' => $tweet_id,
```

Comment:
Optionally these attributes could be put above in `$additional_attributes` (but renamed `$attributes`) and then the `foreach` loop above can just add the `data-` attributes to it. Then here there would be no need for `array_merge()`. It could just be:

```php
$new_node = AMP_DOM_Utils::create_node( $dom, $this->amp_tag, $attributes );
```

Reply:
Done